gulp/config.js

```diff
@@ -63,11 +46,7 @@ module.exports = {
 
         // documentation site
         pldoc_src: pldocSrc + '/static/images',
-        pldoc_src_files: pldocSrc + '/static/images/**/*',
-
-        // example demo
-        demo_src: demoSrc + '/static/images',
-        demo_src_files: demoSrc + '/static/images/**/*'
+        pldoc_src_files: pldocSrc + '/static/images/**/*'
```

Comment:
This is just a +diff because I removed the comma at the end of the line, but yep - we should absolutely be using camelCase for these (or removing them if they're no longer used, which I think these `fonts` ones might not be). I'll see what I can do with cleaning this file up :thumbsup: